corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -234,6 +235,7 @@ def _create_cases(self, datetime_mock, docs=[]):
         return cases
 
     def _create_and_process_changes(self, docs=[]):
+        self.pillow = _get_pillow([self.config], processor_chunk_size=100)
```

Comment:
This is the only one that confuses me. I am not sure why it is necessary to instantiate a new pillow here, but the new config was not propagating to the pillow otherwise in `test_invalid_data_bulk_processor` and `test_invalid_data_serial_processor`